src/pkg/cli/new.go

```diff
@@ -63,6 +64,9 @@ func InitFromSamples(ctx context.Context, dir string, names []string) error {
 	defer tarball.Close()
 	tarReader := tar.NewReader(tarball)
 	term.Info("Writing files to disk...")
+	if err := os.MkdirAll(dir, 0755); err != nil {
```

Comment:
I think we don't need this one if we move the `MkdirAll` on line 84 outside of its `if`

Reply:
Good point